src/adapter/src/catalog/storage.rs

```diff
@@ -416,156 +416,7 @@ async fn migrate(
         // Introduced in v0.47.0
         //
         // TODO(jkosh44) Can be cleared (patched to be empty) in v0.50.0
-        |txn: &mut Transaction<'_>, now, _bootstrap_args| {
```

Comment:
I removed this to get rid of the default_owner role, so the mapping from role id to role was more obvious.